src/api/app/components/bs_request_history_element_component.html.haml

```diff
@@ -1,16 +1,15 @@
-.row
-  .col-auto.pr-0
-    = helpers.image_tag_for(element.user, size: 36, custom_class: 'rounded-circle bg-light border border-gray-400 avatars-counter')
-  .col-10
-    %p
-      = icon
-      = link_to(helpers.realname_with_login(element.user), user_path(element.user))
-      - if element.instance_of?(HistoryElement::RequestSuperseded)
-        superseded this request with
-        = link_to("request ##{element.description_extension}", request_show_path(element.description_extension))
-      - else
-        = element.user_action
-      = link_to("#status-history-#{element.id}", title: l(element.created_at.utc), name: "status-history-#{element.id}") do
-        #{time_ago_in_words(element.created_at)} ago
+.d-inline-flex
+  = helpers.image_tag_for(element.user, size: 36, custom_class: 'mr-2 rounded-circle bg-light border border-gray-400 avatars-counter')
+  %p
+    = icon
+    = link_to(helpers.realname_with_login(element.user), user_path(element.user))
+    - if element.instance_of?(HistoryElement::RequestSuperseded)
+      superseded this request with
+      = link_to("request ##{element.description_extension}", request_show_path(element.description_extension))
+    - else
+      = element.user_action
+    = link_to("#status-history-#{element.id}", title: l(element.created_at.utc), name: "status-history-#{element.id}") do
+      #{time_ago_in_words(element.created_at)} ago
 
-    = render partial: 'webui/shared/collapsible_text', locals: { text: element.comment }
+.pl-4
+  = render partial: 'webui/shared/collapsible_text', locals: { text: element.comment }
```

Comment:
This is intentional. With the current code, a history element is rendered as such:

![example](https://user-images.githubusercontent.com/1102934/188915070-3f292939-2752-4ea5-8ead-5f3438fa27a2.png)

If we would put this code inside the `.d-inline-flex` element, it would render like this:

![broken](https://user-images.githubusercontent.com/1102934/188914779-db8f3cb3-b25a-41f7-8509-87cfe4229f8f.png)